pkg/api/api.go

```diff
@@ -96,16 +96,3 @@ type VerificationCertificateResponse struct {
 	Certificate string `json:"certificate"`
 	Error       string `json:"error"`
 }
-
-// CoverRequest is a client->server request for simulating traffic to the server.
```

Comment:
Do we need to keep this in and mark as deprecated instead?

Reply:
as far as I know , this isn't in use any anyone yet.